### ISSUE TITLE
Imports blueprint code into app

### ISSUE THREAD

**Issue by ef4** [kiwiupover/ember-cli-surge#155]
This addon imports its blueprint build-time code (which should only ever run in Node) into the consuming app's *runtime* (browser) code. This causes the app to contain a broken module. Normal builds don't notice this, but [Embroider notices](https://github.com/embroider-build/embroider/issues/259).

You can verify in an app by running:

```js
Object.keys(requirejs.entries).filter(k => /surge/.test(k))
```

which will show a result like `your-app/blueprints/surge`. And if you try to `require('your-app/blueprints/surge')`, it throws because the module has invalid imports. Which is how I can be sure nothing is actually using it.

This can be fixed by deleting `app/blueprints/surge.js`.

**Comment by balinterdi** [comment]
I've verified that I got past that Embroider build problem by deleting the file Ed suggests so I've submitted #156 to fix this issue.

**Comment by kiwiupover** [comment]
Thanks @ef4 and @balinterdi closed by #156
